portfolio/src/main/java/com/google/sps/Dao.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps;
+
+/** DAO Interface */
+public interface Dao<T> {
+  public void create(T obj);
```

Comment:
It may be beneficial to have this method return the object. The object won't have an ID until it is created so the expected behavior would be for the returned object to have the ID field set. 
```suggestion
  public T create(T obj);
```